### DISCUSSION TITLE
Can't find any sessions when using as a service

### DISCUSSION THREAD

**Discussion by mybadstudios** [naudio/NAudio#1196] [Q&A] [answered]
Hi. When I use NAudio in my desktop application I am able to find all devices and sessions just fine but when I use it as a Working Service Application it returns the system sounds as the only found session(s).

Am I doing something wrong or is this a known limitation or a bug?

**Comment by mdcdi1315** [comment, marked as answer]
No, if you want to use WASAPI on Windows as a service, you cannot use it to playback audio. It is not a limitation of the library, it is a limitation that Microsoft has stated.

**Comment by markheath** [comment]
Yes, this is a Windows limitation and the same issue as #1147
